extensions/flyway/deployment/src/main/java/io/quarkus/flyway/FlywayProcessor.java

```diff
@@ -144,6 +163,21 @@ private void registerSubstrateResources(BuildProducer<SubstrateResourceBuildItem
         }
     }
 
+    private Set<String> getApplicationMigrationsFromPath(final String location, final URL path)
+            throws IOException, URISyntaxException {
+        return walk(Paths.get(path.toURI()))
+                .filter(Files::isRegularFile)
+                .map(it -> Paths.get(location, it.getFileName().toString()).toString())
```

Comment:
This was preexisting but I'm a bit puzzled by this line. I wonder what it's supposed to do.

Reply:
It construct a Set of all migration files compound of a location and each files presents in this location.
For example: **01_init.sql** and **02_update.sql** presents in the location **db/migration** will produce a Set of String containing : **db/migration/01_init.sql** and **db/migration/02_update.sql**.
![map](https://user-images.githubusercontent.com/5189615/64214251-05e57300-ceb0-11e9-8279-444229744b4e.png)

Reply:
Ah OK, I see, they are only used for native resources registration so you need relative paths.